strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml

```diff
@@ -296,7 +296,9 @@
                             data-store="file-system"
                             layout="npm"
                             type="proxy">
-                    <remote-repository url="https://registry.npmjs.org/"/>
+                    <remote-repository url="https://registry.npmjs.org/">
+                        <npm-remote-repository-configuration last-change-id="0" replicate-url="https://replicate.npmjs.com/"/>
```

Comment:
What is the `last-change-id` for? Shouldn't this perhaps be stored in the database?

Reply:
Okay, thanks!